src/util/job-name-validation.tsx

```diff
@@ -64,3 +65,63 @@ export function NameError(name: string, trans: TranslationBundle): string {
     'Name must contain only letters, numbers, spaces, periods, hyphens, and underscores'
   );
 }
+
+export function MaxRetryAttemptsError(maxRetryAttempts: string, trans: TranslationBundle): string {
+  // Check for blank
+  if (maxRetryAttempts === '') {
```

Comment:
This should be an optional parameter, since existing users editing job definitions will not have these values set.